pkg/reconciler/common/constvars.go

```diff
@@ -17,5 +17,6 @@ limitations under the License.
 package common
 
 const (
+	DefaultStageTimeout = 2
```

Comment:
We should make this configurable as large models as we know takes long time to load, probably we can use the `progressDeadline` setting from Knative configuration

Reply:
I have made them configurable with CM or the annotation in service.